src/qt/peertablemodel.cpp

```diff
@@ -173,9 +177,14 @@ QVariant PeerTableModel::data(const QModelIndex &index, int role) const
             return QString::fromStdString(rec->nodeStats.cleanSubVer);
         case Ping:
             return GUIUtil::formatPingTime(rec->nodeStats.dMinPing);
+        case Sent:
+            return GUIUtil::formatBytes(rec->nodeStats.nSendBytes);
+        case Received:
+            return GUIUtil::formatBytes(rec->nodeStats.nRecvBytes);
         }
     } else if (role == Qt::TextAlignmentRole) {
-        if (index.column() == Ping)
+        // everything after ping will be right aligned
+        if (index.column() >= Ping)
```

Comment:
Explicit is better?
```cpp
switch (index.column()) {
    case Received:
    case Sent:
        return Qt::AlignRight | Qt::AlignVCenter;
    default:
        return QVariant();
}
```
(Remove comment above)

Reply:
Agreed